### PR TITLE
Update Polish for 5.8

### PR DESCRIPTION
**Pull request by BlackusPL**
Changed "Mediów" to "Multimediów" because before it sounds like it was created for Twitter, TV program, etc. Now it sounds like it is for Youtube, Netflix, etc.

### CODE REVIEW

**Comment by dead8309** [comment]
Fix build issue

File: common/resources/src/main/res/values-pl/strings.xml [hunk 1]
```diff
@@ -188,20 +188,20 @@
...
+    <string name="start_appDetection">Uruchom Wykrywanie Aplikacji</string>
+    <string name="start_mediaRPC">Uruchom RPC Multimediów</string>
+    <string name="start_experimentalRPC">Uruchom Eksperymentalne RPC</string>
+    <string name="stop_rpc">Zatrzymaj RPC</string>
+    <string name="stop">Zatrzymaj</string>
+    <string name="start_appDetection_toast">Uruchomiono Wykrywanie Aplikacji</string>
+    <string name="start_mediaRPC_toast">Uruchomiono RPC Multimediów</string>
+    <string name="start_experimentalRPC_toast">Uruchomiono Eksperymentalne RPC</string>
+    <string name="stop_rpc_toast">Zatrzymano wszystkie usługi</string>
+    <string name="show_playback_state">Pokaż stan Playback'u</string>
```
Comment (dead8309) [inline common/resources/src/main/res/values-pl/strings.xml:202]: Escape `'` with `\'`

File: common/resources/src/main/res/values-pl/strings.xml [hunk 2]
```diff
@@ -188,20 +188,20 @@
...
+    <string name="copied_to_clipboard">Skopiowano do schowka</string>
+    <string name="application_id">ID Aplikacji</string>
+    <string name="custom_application_id">ID aplikacji własnej aktywności</string>
+    <string name="custom_application_id_desc">Podmienia domyślne ID Aplikacji Discord'a</string>
```
Comment (dead8309) [inline common/resources/src/main/res/values-pl/strings.xml:206]: Escape `'` with `\'`

**Comment by BlackusPL** [comment]
Now fixed

**Pull request merged**